src/NServiceBus.Core.Tests/Routing/DistributionPolicyTests.cs

```diff
@@ -0,0 +1,54 @@
+﻿namespace NServiceBus.Core.Tests.Routing
+{
+    using System.Collections.Generic;
+    using NServiceBus.Routing;
+    using NUnit.Framework;
+
+    [TestFixture]
+    public class DistributionPolicyTests
+    {
+        [Test]
+        public void When_no_strategy_configured_for_endpoint_should_use_round_robbin_strategy()
```

Comment:
argh

http://www.crossfitcypress.com/wp-content/uploads/2015/08/Batman-And-Robin-Cute.jpg